app/code/Magento/User/Controller/Adminhtml/Auth.php

```diff
@@ -6,31 +6,45 @@
 namespace Magento\User\Controller\Adminhtml;
 
 use Magento\Framework\Encryption\Helper\Security;
+use Magento\Framework\App\ObjectManager;
+use Magento\Backend\App\AbstractAction;
+use Magento\Backend\App\Action\Context;
+use Magento\User\Model\UserFactory;
+use Magento\Backend\Helper\Data;
+use Magento\Framework\Exception\LocalizedException;
 
 /**
  * \Magento\User Auth controller
  */
-abstract class Auth extends \Magento\Backend\App\AbstractAction
+abstract class Auth extends AbstractAction
 {
     /**
      * User model factory
      *
-     * @var \Magento\User\Model\UserFactory
+     * @var UserFactory
      */
     protected $_userFactory;
+    
+    /**
+     * @var Data
+     */
+    protected $_backendDataHelper;
 
     /**
      * Construct
      *
-     * @param \Magento\Backend\App\Action\Context $context
-     * @param \Magento\User\Model\UserFactory $userFactory
+     * @param Context $context
+     * @param UserFactory $userFactory
+     * @param Data $backendDataHelper
      */
     public function __construct(
-        \Magento\Backend\App\Action\Context $context,
-        \Magento\User\Model\UserFactory $userFactory
+        Context $context,
+        UserFactory $userFactory,
+        Data $backendDataHelper = null
     ) {
         parent::__construct($context);
         $this->_userFactory = $userFactory;
+        $this->_backendDataHelper = $backendDataHelper ?: ObjectManager::getInstance()->get(Data::class);
```

Comment:
Please do PR on 2.3-develop and remove the `?: ObjectManager::getInstance()->get(Data::class);` part

Reply:
This part was done correctly, no need to introduce backward incompatible changes if possible